docs/spelling_wordlist.txt

```diff
@@ -5,6 +5,7 @@ authorised
 backend
 backends
 behaviour
+blockdevice
```

Comment:
This doesn't appear to be anywhere in the docs and I've just built spelling without it here and the spelling test passes, so I think it can go